### ISSUE TITLE
Loss is nan on validation

### ISSUE THREAD

**Issue by maiiabocharova** [alankbi/detecto#98]
I have created a custom dataset using labelImg, everything looks fine, but while training I get `Loss: nan` after every epoch.

I reduced the size of the dataset for better manual checking, checked the names of the files, everything is fine, but the model is not training. 

Can you help me?

**Comment by alankbi** [comment]
Do any of the suggestions in #91 and #36 work? This issue occasionally comes up for some people and I'm not sure the best solution besides maybe double-checking/recreating environments to see if that helps.

**Comment by Tanmay-Trantor** [comment]
Just decrease the learning rate, it might getting large gradient descent. In my case I reduced it to 0.001
